village_add_thanks.php

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta property="fb:appid" content="<?php print FACEBOOK_APP_ID; ?>"/>
+<meta property="og:image" content="<?php print PICTURES_DIR.$bannerPicture; ?>"/>  <!-- here could we put the pic uploaded by the user? -->
+<meta property="og:title" content="I just added <?php print $villageName; ?> to Village X's map!"/>
+<meta property="og:url" content="<?php print BASE_URL.$projectId; ?>"/>  <!-- not sure what to put here -->
+<meta property="og:description" content="Village X maps villages and funds development projects chosen by them."/>
+
+<?php 
+$metaProvided = 1; 
+include('header.inc'); ?>
+
+<div class="row" style="height:90vh;">
+
+<div class="col s12 m12 l6" style="height:100%;padding:7% 6% 6% 6%">
+	<div class="valign-wrapper">
+		<div>
+        		<div class="row left-align">
+        			<h3 class="header col s12 black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text" style="font-weight: 300">Thanks, (name), for adding your village in (country) on (date)!</h3>
+        			<h4 class="header col s12 black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text" style="font-weight: 300">We put your village on our map to recognize your commitment to community-led 
+        			development and share your village's development needs with organizations that might be able to help.</h4> 
+        			<h4 class="header col s12 black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text" style="font-weight: 300">Please add more villages to the map and encourage your friends to do the same.</h4>
+        			<h4 class="header col s12 black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text" style="font-weight: 300">Sincerely,</h4>
+        			<h4 class="header col s12 black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text" style="font-weight: 300">Village X Team</h4>
+        		 
+        		</div>
+			<br>
+			<div class="row center">
+				<h5 class="black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text"><b>Share the good news on Facebook and Twitter!</b>
+				</h5>	
+			</div> 
+			<div class="row center">
```

Comment:
These are appearing over the footer image.  There looks to be an issue with the layout somewhere.

Reply:
will take a look

Reply:
i'm at a loss re what's happening here.  very weird.  with the changes you've made (thank you!), the page won't render without a village id. is it possible for you to provide a url with a village id that would allow me to fix the problems you identified?

Reply:
footer image is rendering now

Reply:
![Screen Shot 2019-07-11 at 9 54 08 PM](https://user-images.githubusercontent.com/2419351/61073444-8050c400-a426-11e9-9048-9e77f009a692.png)

Are you seeing this on your end?

Reply:
nope. seeing this.  
<img width="818" alt="Screen Shot 2019-07-11 at 9 12 19 PM" src="https://user-images.githubusercontent.com/30181386/61095243-a70bfc00-a420-11e9-9900-96f2d0722856.png">

Reply:
just did a git pull origin on the branch.  now i'm see this.
<img width="1301" alt="Screen Shot 2019-07-11 at 9 14 20 PM" src="https://user-images.githubusercontent.com/30181386/61095319-ed615b00-a420-11e9-8b9c-116803ab0580.png">

Reply:
<img width="1239" alt="Screen Shot 2019-07-11 at 9 15 17 PM" src="https://user-images.githubusercontent.com/30181386/61095331-023dee80-a421-11e9-912d-824c09858471.png">